legal/Linux-Legal-Metadata.md

```diff
@@ -271,7 +271,7 @@ CMD ["bash"]
 ...
 ```
 
-**debian:buster-slim (local)**
+#### debian:buster-slim (local)
 
 **Docker Metadata**
```

Comment:
Missed this one.

Reply:
```suggestion
Docker Metadata:
```

That was intentional, but I think would be best as not highlighted at all. What do you think?

Reply:
I think the formatting of this entire (local) section is a little strange.  Putting the Docker Metadata in a code block doesn't seem correct because it isn't code.  It is supposed to be a copy of https://github.com/docker-library/repo-info/blob/master/repos/debian/local/buster-slim.md.  i'm not sure what the formatting should be.